pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[build-system]
+# setuptools 61 adds "dynamic" directive support
+requires = ["setuptools >= 61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "changedetection.io"
+description = """\
+    Website change detection and monitoring service, detect changes to \
+    web pages and send alerts/notifications.\
+    """
+readme = {file = "README-pip.md", content-type = "text/markdown"}
+license = {text = "Apache License 2.0"}
```

Comment:
How to include `COMMERCIAL_LICENCE.md` here also? (This licence is in place only to restrict people who want to resell the software, it seems to happen a lot)

Reply:
One can use `license-files` I think.

However, I don't think Apache License 2.0 allows you to restrict who uses it; people can just choose which license they use. You might need to draft a different license or use some copyleft (GPL, etc) to force someone to open source their changes. One thing that has been successful for others is offering a commercial license that includes support from you.

See [libmad](https://www.underbit.com/products/mad/) for an example of using copyleft.

Reply:
Switching to `license-files` as specified in [PEP 639](https://peps.python.org/pep-0639/#add-license-files-key)  needs to wait for pypa/setuptools#3596 to be marked fixed.